src/util/request.js

```diff
@@ -39,9 +39,6 @@ const MISSING_FUNCTION_PARAMETERS = 'Missing parameters for function'
 // By default it would insert `"Accept":"application/json, text/plain, */*"`.
 delete request.defaults.headers.common.Accept
 
-// Enable keepalive for http
-request.defaults.httpAgent = new http.Agent({ keepAlive: true })
```

Comment:
question: was this not the correct way to enable http keep alive?

Reply:
@kleyow Used to be, but no longer works in newer versions of axios.